libraries/AC_PrecLand/AC_PrecLand_Backend.h

```diff
@@ -23,16 +23,22 @@ class AC_PrecLand_Backend
 
     // provides a unit vector towards the target in body frame
     //  returns same as have_los_meas()
-    virtual bool get_los_body(Vector3f& dir_body) = 0;
+    bool get_los_body(Vector3f& dir_body) const {
```

Comment:
maybe we should move this to the .cpp file?

Reply:
yep, I can, I didn't do it as we don't have a .cpp file for now